scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id Serial Primary Key,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  dob date,
+  specialty CHARACTER VARYING
+);
+
+CREATE TABLE patients (
+  id Serial Primary Key,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING,
+  dob date,
+  insurance CHARACTER VARYING
+);
+
+CREATE TABLE appointments (
+  id Serial Primary Key,
+  date TEXT,
+  time TEXT,
```

Comment:
I would steer away from calling these columns `date` and `time`, since those words are also used to represent data types. SQL may or may not throw an error, but it's better to just avoid those names, and use something else (e.g. `appointment_date`) instead.